employees.rb

```diff
@@ -0,0 +1,103 @@
+class Employee
+  attr_accessor :full_name
+  attr_accessor :id
+
+  def initialize(full_name, id)
+    @full_name = full_name
+    @id = id
+  end
+
+  def surname
+    @full_name.split(' ', 2).last
+  end
+
+  def first_name
+    @full_name.split(' ', 2)[0]
+  end
+end
+
+def add_employee(employees)
+  puts '[Add an employee]'
+  print 'Full name: '
+  full_name = gets.chomp
+  print 'ID: '
+  id = gets.chomp
+
+  employee = Employee.new(full_name, id)
+
+  employees << employee
+end
+
+def view_employees(employees)
+  print "Sort by [f]irst name or [l]ast name? "
+  sort_order = gets.chomp
+  sorted_employees(employees, sort_order).each do |employee|
+    puts "#{employee.full_name}, #{employee.id}"
+  end
+end
+
+def sorted_employees(employees, sort_order)
+  employees.sort_by do |employee|
+    if sort_order == "l"
+      employee.surname
+    elsif sort_order == "f"
+      employee.first_name
+    else
+      puts "Wrong letter"
```

Comment:
This will print out `wrong letter` as many times as there are elements in the array. Is there a better way to do this?